core/server/types/kustomization.go

```diff
@@ -55,6 +55,7 @@ func KustomizationToProto(kustomization *kustomizev1.Kustomization, clusterName
 		Suspended:             kustomization.Spec.Suspend,
 		ClusterName:           clusterName,
 		ApiVersion:            version,
+		Tenant:                tenant,
```

Comment:
We are starting to build up a set of fields that exist on every object. I wonder if it is type for them all to "inherit" some base fields so we don't have to change this on every object.